<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>1035</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0" columnstretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="0" column="1" rowspan="2">
       <widget class="QGroupBox" name="inputType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>90</height>
         </size>
        </property>
        <property name="title">
         <string>Input method</string>
        </property>
        <widget class="QRadioButton" name="buttonCamera">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>102</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Camera</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="buttonFile">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>102</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>File</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="plainTextEditVieoFile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160000</width>
          <height>25</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>640</width>
          <height>30</height>
         </size>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>path to the input video path</string>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::NoWrap</enum>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="2">
       <widget class="QLabel" name="labelScreen">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>120</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QTextEdit" name="coutBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>640</width>
          <height>30</height>
         </size>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPlainTextEdit" name="plainTextEditVideoOutput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>640</width>
          <height>30</height>
         </size>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::NoWrap</enum>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1" rowspan="3">
       <widget class="QGroupBox" name="boxControls">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>440</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>170</width>
          <height>700</height>
         </size>
        </property>
        <property name="title">
         <string>Controls</string>
        </property>
        <widget class="QGroupBox" name="boxTrackers">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>140</width>
           <height>151</height>
          </rect>
         </property>
         <property name="title">
          <string>Trackers</string>
         </property>
         <widget class="QRadioButton" name="buttonTrackerOpticalFlow">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>102</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Optical flow</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="buttonTrackerCamShift">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>102</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>CamShift</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="buttonTrackerCompressive">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>102</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Compressive</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="buttonTrackerKyle">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>102</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Kyle</string>
          </property>
         </widget>
        </widget>
        <widget class="QSlider" name="redetectionTime">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>320</y>
           <width>131</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="pageStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>300</y>
           <width>161</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>average redetection time</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="boxDetectors">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>140</width>
           <height>91</height>
          </rect>
         </property>
         <property name="title">
          <string>Detectors</string>
         </property>
         <widget class="QRadioButton" name="buttonDetectorOpencv">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>102</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>OpenCV</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="buttonDetectorKyle">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>102</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Kyle</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="Start">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>360</y>
           <width>114</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Stop">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>390</y>
           <width>114</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
